minio/xml_marshal.py

```diff
@@ -29,9 +29,48 @@
 import io
 
 from xml.etree import ElementTree as s3_xml
+from collections import defaultdict
 
 _S3_NAMESPACE = 'http://s3.amazonaws.com/doc/2006-03-01/'
 
+def xml_to_dict(in_xml):
+    # Converts xml to dict
+    elem = s3_xml.XML(in_xml)
+    return etree_to_dict(elem)
+
+def etree_to_dict(elem):
```

Comment:
This function does not seem to be used

Reply:
This is the same issue with your previous comment.
`xml_to_dict` calls `etree_to_dict` and I am going to call `xml_to_dict` from `get_bucket_encryption`.